beps/0003-auth-architecture-evolution/README.md

```diff
@@ -138,15 +140,18 @@ The `UserInfoService` is exported by `@backstage/auth-node`, and the initial imp
 The `HttpRouterService` interface will be extended with the ability to opt-out of the default protection of endpoints, enabling either cookie auth or unauthenticated access.
 
 ```ts
+export interface HttpRouterServiceAuthPolicy {
+  // The path matches in the same way as if it was passed to `express.Router.use(path, ...)`
+  path: string;
+  allow: 'unauthenticated' | 'user-cookie';
```

Comment:
👀  - this is different from the experimental design, using `'user-cookie'` instead to have it align with the credential type.